installing_on_kubernetes/_topics/migrate_to_appliance.md

```diff
@@ -0,0 +1,102 @@
+## Migrating from Kubernetes to Appliances
+
+### Prerequisites
+1. Deploy an appliance with a database disk.  It is recommended that the appliance being deployed is the same code version as the kubernetes environment.
+
+1. Configure the hostname and ip address on the appliance.
+
+
+### Collect data from the Kubernetes environment
+1. Disable the running environment to prepare for migration
+   ```bash
+   $ oc patch deployment orchestrator -p '{"spec":{"template":{"spec":{"containers":[{"name":"orchestrator","args": ["-c", "sleep 1d"], "command":["/bin/sh"]}]}}}}'
+   ```
+
+1. Take a backup of the database
+   ```bash
+   # Check for enough free disk space
+   $ psql -d vmdb_production -c "SELECT pg_size_pretty(pg_database_size('vmdb_production'));"
+   $ pg_dump -Fc -d vmdb_production > /var/lib/pgsql/data/postgres.dump
+   # Copy the database backup to the appliance (usually under /tmp/ or somewhere that has enough space)
+   ```
+
+1. Extraxt the database password, encryption key, server GUID and region number
+   ```bash
+   $ oc rsh deploy/orchestrator
+   sh-5.1$ cd /var/www/miq/vmdb/
+   sh-5.1$ source ./container_env
```

Comment:
Do we need to `source ./container_env` if we're just cat'ing files below?  Maybe you loaded rails console previously and forgot to remove this step.